index.js

```diff
@@ -0,0 +1,201 @@
+//Task №1
+
+const videos = [
+	{
+	id: 65432445,
+	title: 'The Chamber',
+	},
+	{
+		id: 675465,
+		title: 'Fracture',
+	},
+	{
+		id: 70111470,
+		title: 'Bad Boys',
+	},
+	{
+		id: 654356453,
+		title: 'Die Hard',
+	},
+];
+
+const videoTitleById = (videoItem) => videoItem.reduce((result, video) => {
+    return {
+      ...result,
+      [video.id]: video.title,
+    }
+  }, {});
+  
+console.log(videoTitleById(videos));
+
+
+//Task №2
+
+const boxarts = [
+  {
+    width: 200,
+    height: 200,
+    url: "https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Object/values",
+  },
+  {
+    width: 150,
+    height: 200,
+    url: "https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Object/toString",
+  },
+  {
+    width: 500,
+    height: 200,
+    url: "https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Object/assign",
+  },
+  {
+    width: 425,
+    height: 150,
+    url: "https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Object/valueOf",
+  },
+];
+
+//solutionNumberOne
+const arrayWithArea = boxarts.map((item) => {
+  return {
+    area : item.width * item.height,
+    url : item.url,
+  }
+});
+
+const getUrl = arrayWithArea.reduce((result, item) => {
+  return result.area > item.area ? result : item
+});
```

Comment:
you need to create a function, and work with args, instead of working with variables